sonic-xcvrd/xcvrd/sff_mgr.py

```diff
@@ -435,6 +435,8 @@ def task_worker(self):
                     # Skip if these essential routines are not available
                     continue
 
+                sfp.set_lpmode(False)
```

Comment:
@peterbailey-arista I would suggest to use sfp.set_lpmode() only for SFPs that follow SFF8472. All other transceivers like QSFP+, QSFP28 can support lpmode via EEPROM write. The above code expects each platform to implement `set_lpmode()` even thought that is NOT required for QSFP based modules.
```
if (SFP type module) {
    sfp.set_lpmode(False)
} else {
    api = set_lpmode(False)
}
```

Reply:
After discussing with @byu343 I ended up wrapping api.set_lpmode in a try except instead. Please let me know if this update works for you. Thanks!